src/cli-util.js

```diff
@@ -61,32 +61,35 @@ function getIntOption(argv, optionName) {
     return Number(value);
   }
 
-  console.error(
+  throw new Error(
```

Comment:
Should these be `TypeError`s?

https://nodejs.org/api/errors.html#errors_class_typeerror

Reply:
Yes they should :)